content/2022-06-github-automation-quarkus-jnation/1-quarkus-github-bot.md

```diff
@@ -5,15 +5,14 @@
 ## Triaging
 
 * Constant flow of issues/pull requests
-* A lot of components with different maintainers
+* Many components with different maintainers
 * Need to ping appropriate people
 
 -
 
 ## Affecting milestones
 
 * ~ 200 issues / pull requests without milestone when releasing
-* So much fun affecting the milestone manually for each of them
```

Comment:
This can simply be a comment while talking, I'm sure you won't forget about it :)